UnityProject/Assets/Scripts/Chemistry/Reaction.cs

```diff
@@ -33,6 +34,12 @@ public bool Apply(MonoBehaviour sender, ReagentMix reagentMix)
 			{
 				return false;
 			}
+
+
+			if (!ingredients.All(ingredient => reagentMix[ingredient.Key] >= ingredient.Value) && useExactAmounts == true)
+			{
+				return false;
+			}
```

Comment:
This will use inexact amounts for when there is more of an ingredient than is strictly needed.